src/qt/clientmodel.h

```diff
@@ -69,6 +69,9 @@ class ClientModel : public QObject
     //! Return true if core is doing initial block download
     bool inInitialBlockDownload() const;
     //! Return true if core is importing blocks
+    bool IsImporting() const { return cachedImporting; }
+    //! Return true if it's reindexing.
+    bool IsReindexing() const { return cachedReindexing; }
```

Comment:
It seems we are not using these functions anywhere.

Reply:
Not here, have another branch using them. Will move the commit there.

Reply:
done.